src/components/dock/dock-button/dock-button.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import { h } from '@stencil/core';
+import { newSpecPage, SpecPage } from '@stencil/core/testing';
+import { DockItem } from '../dock.types';
+import { DockButton } from './dock-button';
+
+let page: SpecPage;
+let item: DockItem;
+let eventSpy: jest.Mock;
```

Comment:
No need to have this as a global (that was implicit in my suggestion to move it into the test)